### PR TITLE
[#241 | content] Added Levent Gurses talk on WC & Polymer

### PR DESCRIPTION
**Pull request by Nevraeka**
Presentation content from Web Components DC

### CODE REVIEW

File: src/documents/presentations/webcomponentsdc.html.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<div class="iframe-wrap">
+    <iframe src="//www.youtube.com/embed/1CYeDli9n1U" itemprop="video"></iframe>
+</div>
+
+### Slides
+
+<div class="iframe-wrap">
+  <iframe src="//www.slideshare.net/slideshow/embed_code/44503159" width="425" height="355"
+  frameborder="0" marginwidth="0" marginheight="0" scrolling="no"
```
Comment (zenorocha) [inline src/documents/presentations/webcomponentsdc.html.md]: Unnecessary attributes, only `src` is needed.

File: authors.js [hunk 2]
```diff
@@ -139,6 +139,13 @@ module.exports = {
         gplus: "+joshuapeek",
         twitter: "joshpeek"
     },
+    levent_gurses: {
+        name: "Levent Gurses",
+        bio: "Mobile developer, entrepreneur, speaker, & community organizer",
```
Comment (zenorocha) [inline authors.js]: Please follow the convention:
> "Mark is a senior UI developer..."

**Comment by Nevraeka** [comment]
thank @zenorocha

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Thank you, pull request merged! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Thanks @Nevraeka!

Please check commits above for future contributions.

**Pull request merged**